e.ProcessState.ExitCode undefined

**Describe the bug**
Error while executing `go get` command :
```
Step 10/10 : RUN go get github.com/github/git-lfs
 ---> Running in 061a6da707d3
# github.com/git-lfs/git-lfs/git
go/src/github.com/git-lfs/git-lfs/git/git.go:611:58: e.ProcessState.ExitCode undefined (type *os.ProcessState has no field or method ExitCode)
The command '/bin/sh -c go get github.com/github/git-lfs' returned a non-zero code: 2
```

**To Reproduce**
Steps to reproduce the behavior:

Consider following `Dockerfile`:
```
FROM ubuntu:16.04

ENV DEBIAN_FRONTEND=noninteractive

RUN apt update
RUN apt install -y software-properties-common git
RUN add-apt-repository ppa:gophers/archive
RUN apt update
RUN apt -y install golang-1.9-go
ENV GOPATH=$HOME/go
ENV PATH=$PATH:/usr/lib/go-1.9/bin:$GOPATH/bin

RUN go get github.com/github/git-lfs
```
Run `docker image build . --no-cache`

**System environment**
Dockerized Ubuntu 16.04 on macOS, go 1.9/1.11.

**Additional context**
[Line in a recent commit that causes this issue](https://github.com/git-lfs/git-lfs/commit/686939a3abf72e5889a2e679dcf356bece0f215c#diff-291ad9ab3ec4a8486721c1063fc4e22bR611)

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -18,6 +18,18 @@ jobs:
       with:
         name: ${{ matrix.os }}
         path: bin/assets
+  build-go:
+    name: Build with specific Go
+    strategy:
+      matrix:
+        go: ['1.12.x', '1.13.x']
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go }}
+    - run: script/cibuild
   build-windows:
     name: Build on Windows
     runs-on: windows-latest

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -99,7 +99,9 @@ Here are some tips for getting your question answered as quickly as possible:
 
 ### Prerequisites
 
-Git LFS depends on having a working Go 1.11.0+ environment.
+Git LFS depends on having a working Go development environment.  We officially
+support the latest version of Go, although we try not to break backwards
+compatibility with older versions if it's possible to avoid doing so.
 
 On RHEL etc. e.g. Red Hat Enterprise Linux Server release 7.2 (Maipo), you will neet the minimum packages installed to build Git LFS:
 

